start.py

```diff
@@ -218,7 +225,7 @@ def cm_server_not_dead(primary_node):
                              '/dfs*/dn/current/*'])
 
     logger.info('Restarting Cloudera Manager agents ...')
-    _restart_cm_agents(nodes)
+    _restart_cm_agents(nodes, cdh_version_tuple >= EARLIEST_CDH_VERSION_WITH_CENTOS7)
```

Comment:
Once the nodes are actually up, it may be better to check directly if the container has a CentOS 7 distro than checking it based on CDH version. Same feedback below in any place where we do a check like this since I worry it could potentially be a little flakier than directly inspecting the running container via `execute` command output.

Reply:
Done.

Reply:
Done.